webview/src/webview_common.cpp

```diff
@@ -128,7 +128,24 @@ static int Destroy(lua_State* L)
     return 1;
 }
 
-void ParseOptions(lua_State* L, int argumentindex, RequestInfo* requestinfo)
+
+void ParseHeaders(lua_State* L, int argumentindex, int webview_id)
+{
+    luaL_checktype(L, argumentindex, LUA_TTABLE);
+    lua_pushvalue(L, argumentindex);
+    lua_pushnil(L);
+    while (lua_next(L, -2))
+    {
+        const char* header = lua_tostring(L, -2);
+        const char* value = lua_tostring(L, -1);
+        Platform_AddHeader(L, webview_id, header, value);
```

Comment:
My memory is a bit fuzzy, but I wanted to make sure that the function isn't accidentally converting a table's values to strings.

See documentation here:
https://www.lua.org/manual/5.1/manual.html#lua_tolstring

Reply:
HTTP request values are always strings:

* https://docs.oracle.com/javase/8/docs/api/java/net/URLConnection.html#setRequestProperty-java.lang.String-java.lang.String-
* https://developer.apple.com/documentation/foundation/urlrequest/2011447-setvalue

Reply:
I know, but that doesn't really stop the _user_ to pass in an integer.
And, in that case, is the table modified?

Reply:
The Lua table is not modified. Each header field and value is parsed from the Lua table and a call to addHeader() is made, which will add the header to a Map (Android) or NSMutableDictionary (iOS/macOS) for later use.

Reply:
Ah, got it. I read the docs for lua_tolstring():

If the value is a number, then [lua_tolstring](https://github.com/defold/extension-webview/pull/35#lua_tolstring) also changes the actual value in the stack to a string. (This change confuses [lua_next](https://github.com/defold/extension-webview/pull/35#lua_next) when [lua_tolstring]few(https://github.com/defold/extension-webview/pull/35#lua_tolstring) is applied to keys during a table traversal.)